File: testprojects/scala_main/pipelines/DONOT_OPEN_SCALA/code/src/main/scala/org/main/scla_dep_mgmt/graph/Filter_2.scala
package org.main.scla_dep_mgmt.graph

import io.prophecy.libs._
import org.main.scla_dep_mgmt.config.ConfigStore._
import org.main.scla_dep_mgmt.udfs.UDFs._
import org.main.scla_dep_mgmt.udfs._
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import java.time._

object Filter_2 {

  def apply(spark: SparkSession, in: DataFrame): DataFrame =
    in.filter(col("customer_id") > lit(5))

}

File: testprojects/master_python_mainone/pipelines/PYTHON_SUBGRAPH_CREATOR/code/job/graph/RecSG/Subgraph_1/Subgraph_2/Filter_2.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from prophecy.libs import typed_lit
from job.config.ConfigStore import *
from job.udfs.UDFs import *

def Filter_2(spark: SparkSession, in0: DataFrame) -> DataFrame:
    return in0.filter((col("`c  - int`") > lit(10)))
